starter-code/scripts/models/article.js

```diff
@@ -40,31 +40,38 @@ Article.loadAll = function(inputData) {
 };
 
 /* This function below will retrieve the data from either a local or remote
- source, process it, then hand off control to the View: */
+source, process it, then hand off control to the View: */
 Article.fetchAll = function() {
   if (localStorage.blogArticles) {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    Article.loadAll(JSON.parse(localStorage.blogArticles));
+    articleView.renderIndexPage();
   } else {
     /* Without our localStorage in memory, we need to:
     1. Retrieve our JSON file with $.getJSON
-      1.a Load our json data
-      1.b Store that data in localStorage so that we can skip the server call next time,
-      1.c And then render the index page.*/
+    1.a Load our json data
+    1.b Store that data in localStorage so that we can skip the server call next time,
+    1.c And then render the index page.*/
+    $.getJSON('data/blogArticles.json', function (data) {
+      localStorage.blogArticles = JSON.stringify(data);
+      Article.loadAll(JSON.parse(localStorage.blogArticles));
```

Comment:
Is there something equivalent to the result of parsing local storage that we could place inside of `loadAll()` here?